Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (1.9.1)
-      rubocop (= 1.26.1)
-      rubocop-performance (= 1.13.3)
-    stimulus-rails (1.0.4)
+    standard (1.16.1)
+      rubocop (= 1.35.1)
+      rubocop-performance (= 1.14.3)
+    stimulus-rails (1.1.0)
```

Comment:
### [stimulus-rails](https://stimulus.hotwired.dev)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.0.4 → 1.1.0

[_~~change-log~~, [source-code](https://github.com/hotwired/stimulus-rails), [gem-diff](https://my.diffend.io/gems/stimulus-rails/1.0.4/1.1.0)_]





<details>
<summary>Commits</summary>

A change of **5** commits. See the full changes on [the compare page](https://github.com/hotwired/stimulus-rails/compare/v1.0.4...v1.1.0).

These are the individual commits:
- (0849a02) [Add Rails 7 to the CI matrix.  Adds a matrix for Rubies cove…](https://github.com/hotwired/stimulus-rails/commit/0849a02be09e91f99d8a4dd85ea328ce6691f3c5)
- (25f8fce) [Precompile assets constant for opt out possibility (#96)](https://github.com/hotwired/stimulus-rails/commit/25f8fceea24087da4f7d4a66c5942d73fc2e771d)
- (26754fe) [Remove `.js` extension from JavaScript imports (#104)](https://github.com/hotwired/stimulus-rails/commit/26754fe0a4fa0893f191521e93a7e89592f78a0a)
- (55113d2) [Bump for Stimulus 3.1.0](https://github.com/hotwired/stimulus-rails/commit/55113d21e3f17a34cebe9263797c671e02b1445f)
- (abbab9a) [Bump version for 1.1.0](https://github.com/hotwired/stimulus-rails/commit/abbab9aacf86e44981a7d4764d2955e76d74636e)


</details>